mappings/net/minecraft/structure/WoodlandMansionGenerator.mapping

```diff
@@ -51,30 +53,105 @@ CLASS net/minecraft/unmapped/C_dizzctht net/minecraft/structure/WoodlandMansionG
 			ARG 0 identifier
 	CLASS C_qhxyqjoa SecondFloorRoomPool
 	CLASS C_qibwxtzr MansionParameters
+		FIELD f_bgopczqz CLEAR I
```

Comment:
```suggestion
		FIELD f_bgopczqz CLEARING I
```
?

Reply:
It's used as a value for a grid to set the piece type
```java
private static final int CLEAR = 0;
private static final int CORRIDOR = 1;
private static final int ROOM = 2;
private static final int START_ROOM = 3;
private static final int TEST_ROOM = 4;
private static final int BLOCKED = 5;
```